### PR TITLE
Let users launch NextFlow from file browser

### LINKED ISSUE #469: [MacCoss] Let users choose NextFlow config
- [x] Choose from multiple configs in directory
- [x] Launch job from file browser
- [x] Substitute URL for selected directory
- [x] Expose results 
- [x] Merge 📌

### PR DESCRIPTION
**Pull request by labkey-jeckels**
#### Rationale
This is the next step in making NextFlow functional for production use.

#### Related Pull Requests
* https://github.com/LabKey/platform/pull/6136

#### Changes
* Launch jobs via the file browser
* Let users choose from multiple config files
* Substitute into the config file the URL of the data files
* Create an experiment run with the inputs and outputs of the job

### CODE REVIEW

File: nextflow/build.gradle [hunk 1]
```diff
@@ -6,5 +6,10 @@ plugins {
 
 dependencies {
     BuildUtils.addLabKeyDependency(project: project, config: "modules", depProjectPath: BuildUtils.getPlatformModuleProjectPath(project.gradle, "pipeline"), depProjectConfig: "published", depExtension: "module")
-}
 
+    compileOnly "org.projectlombok:lombok:${lombokVersion}"
+    annotationProcessor "org.projectlombok:lombok:${lombokVersion}"
+
+    BuildUtils.addLabKeyDependency(project: project, config: "modules", depProjectPath: BuildUtils.getPlatformModuleProjectPath(project.gradle, "experiment"), depProjectConfig: "published", depExtension: "module")
+    BuildUtils.addLabKeyDependency(project: project, config: "modules", depProjectPath: BuildUtils.getPlatformModuleProjectPath(project.gradle, "pipeline"), depProjectConfig: "published", depExtension: "module")
```
Comment (ankurjuneja) [inline nextflow/build.gradle:14]: this dependency on pipeline seems to be repeated (L8)

**Review by ankurjuneja — APPROVED** [review]

File: nextflow/build.gradle (hunk 1 above)
Comment (labkey-jeckels) [reply nextflow/build.gradle:14]: One is the pipeline dependency, the other is the experiment dependency.

Comment (ankurjuneja) [reply nextflow/build.gradle:14]: pipeline dependency is already there on Line 8

Comment (labkey-jeckels) [reply nextflow/build.gradle:14]: Ah, sure enough. Thanks. Removed.

**Pull request merged**